pkg/bootstrap/platform/discovery.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const defaultTimeout = 5 * time.Second
+const Platforms = 3
```

Comment:
NumPlatforms. and not an exported constant as its local to this file

Reply:
Fixed